engine/src/main/java/org/terasology/entitySystem/entity/internal/EngineEntityManager.java

```diff
@@ -109,14 +109,14 @@
      *
      * @param subscriber
      */
-    void subscribe(EntityChangeSubscriber subscriber);
+    void subscribeChangeListener(EntityChangeSubscriber subscriber);
```

Comment:
This doesn't read very well. Perhaps "subscribeToChanges" or "subscribeForChanges". Or just combine the two subscription interfaces.